docs/running-local.md

```diff
@@ -0,0 +1,236 @@
+Running On a Local (Big) Machine
+================================
+
+Prerequisites
+-------------
+
+You need a beefy Linux `x86_64` machine, with `docker` and `docker-compose`
+available (and ready to use by the current UNIX user).
+You'll also need to access this host on port 443 (HTTPS).
+
+For example on Ubuntu 16.04 (LTS):
+
+    sudo apt-get install -y docker docker-compose
+    sudo adduser $USER docker
+    sudo su $USER # so that the group is taken into account
+    docker ps # should not complain :)
+
+Get Secotrec-local
+------------------
+
+Then we get the `secotrec-local` application:
+
+    mkdir ~/bin/
+    wget -O ~/bin/secotrec-local https://storage.googleapis.com/smondet-experiments/secotrec-local-Linux-x86_64-03
+    chmod +x ~/bin/secotrec-local
+    export PATH=~/bin:$PATH
+
+And do a quick test:
+
+    secotrec-local --help
+
+Local Infrastructure Deployment
+-------------------------------
+
+Secotrec-local is configured with environment variables, you can put them in a
+file and source it, or just paste them in your shell:
+
+    # HTTPS port to access the Ketrew UI:
+    export tls_port=443
+    # Directory where Epidisco will write files
+    export biokepi_work=$HOME/biokepi-work/
+    # Long, random, secret string of you choice:
+    export ketrew_auth_token=dsleaijdej308098ddecja9c8jra8cjrf98r
+    # The maximal number of jobs that are allowed to run at once:
+    export coclobas_max_jobs=8
+    # We also want a configured Epidisco development environment:
+    export epidisco_dev=$HOME/epidisco-shared
+    # We want to get recent version of most tools:
+    export coclobas_docker_image=hammerlab/keredofi:coclobas-aws-biokepi-dev
+
+You check that the configuration is taken into account:
+
+    secotrec-local print-conf
+
+You need this set up (the first time, for now):
+
+    mkdir -p $epidisco_dev
+    chmod -R 777 $epidisco_dev
+    mkdir -p /tmp/secotrec-local-shared-temp
+    chmod -R 777 /tmp/secotrec-local-shared-temp
+
+Then you can launch the setup:
+
+    secotrec-local up
+
+the first time it takes a while because it pulls docker images.
+
+Check the status of the deployment with:
+
+    secotrec-local status
+
+Using the hostname or IP address of your host, you can access the Ketrew WebUI
+at (using the auth-token you defined above):
+
+`https:// ... /gui?token=dsleaijdej308098ddecja9c8jra8cjrf98r`
+
+
+Submit a First Epidisco Run
+---------------------------
+
+Let's generate the Biokepi “infrastructure” configuration:
+
+    secotrec-local biokepi-machine $epidisco_dev/biokepi_machine.ml
+
+and then *enter* the docker-container which has all the necessary dependencies
+(incl. Epidisco itself):
```

Comment:
in secotrec-local with the `epidisco_dev` and `tlp_port` options, secotrec creates 5 docker-compose services:

- epidisco-dev: service that does nothing, but is nice to `exec` into like we do here
- pg: database for coclobas and ketrew
- kserver: ketrew server
- tlstun: TLS front-end
- coclo: coclobas server

the user of this tutorial should not need to know any of that, just type the command, enjoy the setup.